core/src/main/scala/scodec/bits/BitVector.scala

```diff
@@ -1013,6 +1016,10 @@ sealed trait BitVector extends BitwiseOperations[BitVector, Long] with Serializa
    */
   final def digest(digest: MessageDigest): BitVector = BitVector(bytes.digest(digest))
 
+  final def cipher(ci: Cipher, key: SecretKeySpec, opmode: Int): BitVector = BitVector(bytes.cipher(ci, key, opmode))
+  final def encrypt(ci: Cipher, key: SecretKeySpec): BitVector = cipher(ci, key, Cipher.ENCRYPT_MODE)
```

Comment:
On other issue -- I think we should support initializing the cipher using all 4 params (opmode, key, algorithm params, and secure random). Not sure what this should look like in the API though. Default arguments is one option. I think this concern is what led to the creation of `CipherFactory` in scodec-core.